torch/csrc/jit/codegen/cuda/ir_base_nodes.cpp

```diff
@@ -105,22 +105,42 @@ class ConstCheck : OptOutConstDispatch {
     is_const_ = is_const_ && b->isConst();
   }
 
+  void handle(const kir::Bool* i) override {
```

Comment:
Seems reasonable, but should we avoid mixing these and just make a copy of isConst somewhere in kir? CC @tlemo

Reply:
I think it's fine for now, but it may not be a bad idea to open an Github issue to revisit it.